setup.py

```diff
@@ -21,7 +21,11 @@
             "pytest==2.9.1",
         ]
     },
-    dependency_links=[],
+    entry_points={
```

Comment:
I think `decl_base` is better. `model_base` could be confused with the base class of the model on its own (as opposed to after it's been made declarative)